doc/sources/installation/installation-windows.rst

```diff
@@ -230,6 +239,36 @@ kivy with git rather than a wheel there are some additional steps:
      export USE_GSTREAMER=1
 
    These variables must be set everytime you recompile kivy.
+   
+   **MSVC:**
```

Comment:
Right, I thought it wasn't used and that you had ordered it so that it would be used.

Virtualenv does indeed do this, it's just that the path is slightly different. To get the correct path in a virtual env you have to do `join(dirname(dirname(sys.executable)), 'include')`. That should fix the problem. So I'd try both  `join(dirname(dirname(sys.executable)), 'include')` and  `join(dirname(sys.executable), 'include')`.